b2sdk/v1/sync.py

```diff
@@ -0,0 +1,126 @@
+        :param exclude_dir_regexes: a tuple of regexes to exclude directories
+        :type exclude_dir_regexes: tuple
+        :param exclude_file_regexes: a tuple of regexes to exclude files
+        :type exclude_file_regexes: tuple
+        :param include_file_regexes: a tuple of regexes to include files
+        :type include_file_regexes: tuple
+        :param exclude_all_symlinks: if True, exclude all symlinks
+        :type exclude_all_symlinks: bool
+        :param exclude_modified_before: optionally exclude file versions modified before (in millis)
+        :type exclude_modified_before: int, optional
+        :param exclude_modified_after: optionally exclude file versions modified after (in millis)
+        :type exclude_modified_after: int, optional
+        """
+        if include_file_regexes and not exclude_file_regexes:
+            raise v2_exception.InvalidArgument(
+                'include_file_regexes',
+                'cannot be used without exclude_file_regexes at the same time'
+            )
+
+        self._exclude_dir_set = v2.RegexSet(exclude_dir_regexes)
+        self._exclude_file_because_of_dir_set = v2.RegexSet(
+            map(v2.convert_dir_regex_to_dir_prefix_regex, exclude_dir_regexes)
+        )
+        self._exclude_file_set = v2.RegexSet(exclude_file_regexes)
+        self._include_file_set = v2.RegexSet(include_file_regexes)
+        self.exclude_all_symlinks = exclude_all_symlinks
+        self._include_mod_time_range = v2.IntegerRange(
+            exclude_modified_before, exclude_modified_after
+        )
+
+
+DEFAULT_SCAN_MANAGER = ScanPoliciesManager()
+
+
+def zip_folders(folder_a, folder_b, reporter, policies_manager=DEFAULT_SCAN_MANAGER):
+    return v2.zip_folders(folder_a, folder_b, reporter, policies_manager=policies_manager)
+
+
+class Synchronizer(v2.Synchronizer):
+    def __init__(
+        self,
+        max_workers,
+        policies_manager=DEFAULT_SCAN_MANAGER,
+        dry_run=False,
+        allow_empty_source=False,
+        newer_file_mode=v2.NewerFileSyncMode.RAISE_ERROR,
+        keep_days_or_delete=v2.KeepOrDeleteMode.NO_DELETE,
+        compare_version_mode=v2.CompareVersionMode.MODTIME,
+        compare_threshold=None,
+        keep_days=None,
+    ):
+        super().__init__(
+            max_workers, policies_manager, dry_run, allow_empty_source, newer_file_mode,
+            keep_days_or_delete, compare_version_mode, compare_threshold, keep_days
+        )
+
+    def make_folder_sync_actions(
+        self,
+        source_folder,
+        dest_folder,
+        now_millis,
+        reporter,
+        policies_manager=DEFAULT_SCAN_MANAGER,
+        encryption_settings_provider=v2.SERVER_DEFAULT_SYNC_ENCRYPTION_SETTINGS_PROVIDER,
+    ):
+        return super().make_folder_sync_actions(
+            source_folder, dest_folder, now_millis, reporter, policies_manager,
+            encryption_settings_provider
+        )
+
+
+class AbstractFolder(v2.AbstractFolder):
```

Comment:
And this class as well. After some digging i see, that the reason is the default argument, but i think it should be mentioned.